SPEC.md

```diff
@@ -490,6 +491,10 @@ The App Container specification defines an HTTP-based metadata service for provi
 The ACE must provide a Metadata service on the address given to the applications via the `AC_METADATA_URL` [environment variable](#execution-environment).
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
+Clients must also authenticate by use of a bearer token.
```

Comment:
@jonboulle I think it'll be more confusing than helpful. This does not implement rfc6750. It's not even OAuth. It just uses the same Authorization header format.